eva/udfs/pytorch_abstract_udf.py

```diff
@@ -112,3 +110,11 @@ def __call__(self, *args, **kwargs):
         if isinstance(frames, pd.DataFrame):
             frames = frames.transpose().values.tolist()[0]
         return nn.Module.__call__(self, frames, **kwargs)
+
+
+class PytorchAbstractTransformationUDF(AbstractTransformationUDF, Compose):
```

Comment:
I am somewhat if `PytorchAbstractTransformationUDF` should subclass torchvision transforms or nn.Module. A neural network can also be used to transform a frame. Let me know what you think.

Reply:
Doesn't `PytorchAbstractUDF` itself inherit from `nn.Module`?